tools/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -651,9 +661,9 @@ class QualificationSuite extends FunSuite with BeforeAndAfterEach with Logging {
     assert(headersDetailed.size == QualOutputWriter
       .getDetailedHeaderStringsAndSizes(Seq(qualApp.aggregateStats.get), false).keys.size)
     assert(valuesDetailed.size == headersDetailed.size)
-    // 5 should be the Gpu Opportunity
-    assert(headersDetailed(5).contains("GPU Opportunity"))
-    assert(valuesDetailed(5).toDouble > 0)
+    // 9 should be the Gpu Opportunity
+    assert(headersDetailed(9).contains("GPU Opportunity"))
+    assert(valuesDetailed(9).toDouble > 0)
```

Comment:
somewhere we should test the output of the new fields is what we expect, choose one of the static event logs we process so we know what the values will be

Reply:
Comparing to the static files will make the test redundant, right?
I modified the test to verify that:

- all headers are correct
- the recommendation field matches the `GPU Speedup`
- `CPU-Percent` is in the correct range [0,100]
- `App-duration` is GTE (`GPU opportunity` and `SQL Duration`)
- For all other numeric fields, the value is GT `0`.